[2024-06-14T08:36:37.102+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-14T08:36:37.121+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: f1_data_pipeline_taskflow_with_kafka.wait_for_elt_dag manual__2024-06-14T08:36:35.437153+00:00 [queued]>
[2024-06-14T08:36:37.130+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: f1_data_pipeline_taskflow_with_kafka.wait_for_elt_dag manual__2024-06-14T08:36:35.437153+00:00 [queued]>
[2024-06-14T08:36:37.130+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-14T08:36:37.142+0000] {taskinstance.py:2330} INFO - Executing <Task(ExternalTaskSensor): wait_for_elt_dag> on 2024-06-14 08:36:35.437153+00:00
[2024-06-14T08:36:37.151+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=698) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-14T08:36:37.154+0000] {standard_task_runner.py:63} INFO - Started process 712 to run task
[2024-06-14T08:36:37.155+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'f1_data_pipeline_taskflow_with_kafka', 'wait_for_elt_dag', 'manual__2024-06-14T08:36:35.437153+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/kafka_data_scrape.py', '--cfg-path', '/tmp/tmpv5whgb7a']
[2024-06-14T08:36:37.160+0000] {standard_task_runner.py:91} INFO - Job 44: Subtask wait_for_elt_dag
[2024-06-14T08:36:37.214+0000] {task_command.py:426} INFO - Running <TaskInstance: f1_data_pipeline_taskflow_with_kafka.wait_for_elt_dag manual__2024-06-14T08:36:35.437153+00:00 [running]> on host 02396e2d7ca5
[2024-06-14T08:36:37.290+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='f1_data_pipeline_taskflow_with_kafka' AIRFLOW_CTX_TASK_ID='wait_for_elt_dag' AIRFLOW_CTX_EXECUTION_DATE='2024-06-14T08:36:35.437153+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-14T08:36:35.437153+00:00'
[2024-06-14T08:36:37.291+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-14T08:36:37.291+0000] {baseoperator.py:399} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2024-06-14T08:36:37.291+0000] {external_task.py:258} INFO - Poking for tasks ['load_raceResults_task'] in dag f1_data_pipeline_taskflow on 2024-06-14T08:36:35.437153+00:00 ... 
[2024-06-14T08:36:37.292+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/session.py:79: DeprecationWarning: This method is deprecated and will be removed in future.
  return func(*args, session=session, **kwargs)

[2024-06-14T08:37:37.299+0000] {external_task.py:258} INFO - Poking for tasks ['load_raceResults_task'] in dag f1_data_pipeline_taskflow on 2024-06-14T08:36:35.437153+00:00 ... 
[2024-06-14T08:38:37.311+0000] {external_task.py:258} INFO - Poking for tasks ['load_raceResults_task'] in dag f1_data_pipeline_taskflow on 2024-06-14T08:36:35.437153+00:00 ... 
[2024-06-14T08:39:28.786+0000] {local_task_job_runner.py:310} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-06-14T08:39:28.787+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-14T08:39:28.788+0000] {process_utils.py:132} INFO - Sending 15 to group 712. PIDs of all processes in the group: [712]
[2024-06-14T08:39:28.788+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 712
[2024-06-14T08:39:28.788+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-06-14T08:39:28.789+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-14T08:39:28.815+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=712, status='terminated', exitcode=0, started='08:36:36') (712) terminated with exit code 0
